### ISSUE TITLE
[Усе контэнт тайпы] Паправіць адлюстроўванне маркірованых і нумераваных спісаў ва ўнутраных.

### ISSUE THREAD

**Issue by Omeljusik** [falanster/devfalanster#83]
https://ftest.falanster.by/ru/node/1164 
У рэлагацыйнай панэлі спіс як быццам ствараецца, але не адлюстроўваецца на старонке. 
![image](https://user-images.githubusercontent.com/32821205/61460664-de762d80-a977-11e9-9dd4-03cba36d11bf.png)

З лічбавымі спісамі дарэчы ўсё добра. Не працуюць менавіта маркірованыя.

**Comment by koverchik** [comment]
@valeria Где-нибудь в задаче это прописано? Как эти списки должны выглядеть?

**Comment by Omeljusik** [comment]
@koverchik нет, было вырашына, што пакідаем па дэфолту, як гэта ёсць у рэдактаре.

Калі патрэбен нейкі варыянт, то менавіта так, як зараз гэтыя маркеры адлюстроўваюцца ў рэлагацыйнай панэлі, калі ты ствараеш спіс (кропачкі). 
Але гэты спіс не адлюстроўваецца на апублікаванай старонке.

**Comment by koverchik** [comment]
@Omeljusik по дефолту выводится у нас без маркеров.

**Comment by Omeljusik** [comment]
## Зрабіць адлегласць унутар ад левага края асноўнага тэкста для маркированых спісаў такую ж, якая зараз для нумераваных. 
https://ftest.falanster.by/be/news/test-vyvada-navin-u-blog-i-na-galounay

![image](https://user-images.githubusercontent.com/32821205/62826190-e6fd0500-bbbf-11e9-80b9-04c26bbefd47.png)
